src/coord/src/coord.rs

```diff
@@ -166,6 +166,8 @@ pub struct Coordinator {
     symbiosis: Option<symbiosis::Postgres>,
     /// Maps (global Id of arrangement) -> (frontier information)
     indexes: ArrangementFrontiers<Timestamp>,
+    /// Map of frontier information for sources
+    sources: ArrangementFrontiers<Timestamp>,
```

Comment:
I feel weird using this type for sources but for now I think this type kind of naturally does what we want

Reply:
Is there a reason not to have it in `indexes` and rename that to idk `frontiers`?

Reply:
`DataflowBuilder` specifically uses `indexes` as a place to find information about only indexes and I wasn't sure how mandatory all of that was